Tests/DataPipeline/APITest.swift

```diff
@@ -1,5 +1,4 @@
 import CioDataPipelines // do not use `@testable` so we can test functions are made public and not `internal`.
-import CioInternalCommon
```

Comment:
now we can find out missing aliases this way as well

Reply:
> this way as well
Thanks for adding the comment but wondering what "this way" is ? 🤔

Reply:
By removing the `CioCommon` from the test class. It would give an error if we utilize something from common but isn't part of alias